CONTRIBUTING.md

```diff
@@ -29,6 +29,48 @@ To contribute a change, please:
 6. If changes are requested, the PR author should address the comments, and then
    ask for review again.
 
+### Developing the cookiecutter template
+
+We quite like installing cookiecutter as a uv tool:
+
+```sh
+uv tool install cookiecutter
+```
+
+To run cookiecutter using a specific branch of the template:
+
+```sh
+cookiecutter gh:UCL-ARC/python-tooling --checkout <branch-name>
+```
+
+If you have this repo locally (presumably the case if you're developing), you
+can run the following:
+
+```sh
+cookiecutter /path/to/your/checkout/of/python-tooling --checkout <branch-name>
+```
+
+Perhaps obvious, you can omit the `--checkout` option if you're already on the
+branch you want to test.
+
+### Developing the recommended tooling pages
+
+Pages all live in the
+[docs/pages](https://github.com/UCL-ARC/python-tooling/tree/main/docs/pages)
+sub-directory, and are written in markdown.
+
+To build the webpage locally (for testing)
+
+1. [Install jekyll]
```

Comment:
This syntax is wild

Reply:
Do you have a concrete alternative suggestion?

Reply:
```suggestion
1. [Install jekyll](https://jekyllrb.com/docs/installation)
```

Reply:
Meh. Consistency with the rest of the file? Everywhere else we're using a list of links. So we should make this a policy and have a linter which enforces it.

Reply:
This is the only repo I've used that follows this syntax, so takes me time to get used to